documentation/asset-tracker/topics/how-assettracker-works.md

```diff
@@ -0,0 +1,196 @@
+(how-assettracker-works-doc)=
+
+# How AssetTracker works
+
+AssetTracker builds on Hyperview's capabilities by adding specialized hardware and software to achieve automated Rack U-level tracking across one or more data centers.
+
+## Terminology
+
+- **Gateway**, this is the device that handles communication between the hardware and the data collector.
```

Comment:
I am still seeing inconsistencies with the casing of "Gateway".

Reply:
Fixed, thanks!